server/index.js

```diff
@@ -1,29 +1,80 @@
 var express = require('express');
+var app = express();
+var server = require('http').createServer(app);
+
 var bodyParser = require('body-parser');
-// UNCOMMENT THE DATABASE YOU'D LIKE TO USE
-// var items = require('../database-mysql');
-// var items = require('../database-mongo');
+var helpers = require('./helper-functions');
 
-var app = express();
 
-// UNCOMMENT FOR REACT
```

Comment:
We should drop all of the advisory comments, right? Also, we have settled on React.